\chapter{Results}

% Characterizing flux differences 
% (Key figures: e.g. scatter plots, simulated flux maps, cofactor pie plots)


% Results Section 1. a figure panel with 
% - A: brief metabolic map; 
% - B: exchange fluxes;
% - C: main intracellular fluxes. Different GRs on x axis


\textit{Rhodotorula toruloides} can naturally accumulate high amounts of lipids, but the 
metabolic principles that make this possible and differentiate, if true, \textit{R. toruloides} from other oleaginous and non-oleaginous yeast are not fully understood.
Biosynthesis of the main lipid precursors acetyl-CoA and NADPH takes place in the central carbon metabolism. A better understanding of which 
metabolic pathways are used in production of these precursors and thus contribute to lipid accumulation, would aid in designing better 
metabolic engineering strategies for increasing lipid production.

Genome-scale metabolic models contain all known biochemical reactions of the specific organism and allow the calculation of
metabolic fluxes, which represent the activity of metabolic pathways under specified conditions. 
This makes GEMs important tool for studying metabolism, but it is important that the predictive power of the GEM is
adequate. For \textit{R. toruloides} several genome-scale metabolic models are available, but so far comprehensive overview of 
simulations focused on central carbon metabolism with these models has not been presented.
This work will help in the future to design \textit{R. toruloides} as the leading microbial cell factory for production of microbial oils.


% \section{Intracellular fluxes and supply of NADPH} %


% Describe the fluxes in central carbon metabolism of R. toruloides;
%% Explain the preference towards XPK or ACL pathway for producing acetyl-CoA

% Extract cofactor balances and analyze what makes model to prefer one or the other.
% Explain where the NADPH is produced  

% For plotting, the enzymes that had fluxes lower than 2% were added to the 'Others' section


% Also cofactor balances of NADH and ATP were plotted.
% * For the model rhto-GEM, there is no difference between any cofactor balances on different growth rates nor objective function
% * iRhtoC - same as with rhto-GEM
% * IFO0880 - differences between the objective functions on the lowest growth rate: biomass max predicts alcohol dehydrogenase as the main source of NADPH (a loop??), NGAM min predicts Homoserine dehydrogenase
%  on the highest GR both obj functions predict alcohol dehydrogenase as the main source, consuming fluxes differ slightly on both GRs - but for us it's not important?
% NADH and ATP fluxes don't differ
% * IFO0880_jsb - NADPH: on lowest GR both obj functions predict fairly similarly that aldehyde dehydrogenase is the main source of NADPH
% on highest GR biomass max as obj predicts alcohol dehydrogenase and NGAM min predicts homoserine dehydrogenase
% NADH fluxes don't differ on different GR rates but they differ between biomass max and NGAM min as obj:
%  biomass max - predicts several enzymes, NGAM min redicts glyceraldehyde-phosphate dehydrogenase as the main source of NADH
% ATP don't differ between GR but between objective functions: biomass max predicts ATP synthase as the main source and NGAM min ADP/ATP transporter

% Differences between models:
% 

% \textbf{Biomass maximisation as objective function}

% % Solution fluxes of rhto-GEM showed that NADPH is recycled via ... . 
% Production and consumption of NADPH with the different models is shown in figures below.


% Four genome-scale metabolic models of \textit{Rhodotorula toruloides} - rhto-GEM, iRhto1108, Rt\_IFO0880 and Rt\_IFO0880\_LEBp2023 - 
% were used for simulating this yeast's metabolic flux distribution using flux balance analysis. 
% Different objective functions, maximisation of biomass and minimisation
% of NGAM, were tested under carbon limitation. 


% Fluxes through notable phosphoketolase and ATP-citrate lyase pathways, 
% that are known to be important for production of lipid precursors in olegeanous yeast, were in the center of interest and are distinguished in intracellular 
% flux figures from other enzymes using dashed line.
% What is more, lipid synthesis demands high amounts of NADPH, but only a few enzymes can generate it. 
% Pentose phosphate pathway and malic enzyme have been proposed as the main candidate enzymes for recycling NADPH in \textit{R. toruloides} 
% \cite{Ratledge2014}. Main intracellular fluxes of central carbon metabolism and production and consumption of NADPH were compared between the simulations 
% of four models. 


\section{Biomass maximisation as an objective function}

Firstly, simulations with biomass maximization as an objective function were carried out. The solutions were constrained over five experimental glucose uptake rates - $0.476, 1.114, 1.648, 2.305$ and $3.1$ \unit{mmol/gDW/h} (Table \ref{table:LabData}).
All simulated fluxes are available on a Github repository (\url{https://github.com/maivehanni/BSc_thesis/tree/main/All_simulated_fluxes}). 
In further analysis, selected exchange and intracellular fluxes were in the focus, and they are visualized over the biomass growth rate and shown below.

To explore the intracellular flux patterns over increasing growth rates, especially the pathways that generate acetyl coenzyme A in glucose catabolic pathways, all figures show the fluxes over growth rates from $0.05$ to $0.25$ \unit{1/h}, that is the range predicted by the models, when constrained over experimentally measured specific glucose uptakes.
For the investigation of the predicted source of NADPH regeneration in metabolism, NADPH producing and consuming fluxes were visualized on pie charts on growth rates $0.05 - 0.25$ \unit{1/h}, but only the ones that significantly differed on different growth rates, are shown below. 

\textbf{rhto-GEM}
% Here, you should briefly describe other intracellular fluxes plotted as well. This goes like, “at the x5p branching point, the flux through TKT
% and TAL was predicted to be increased linearly from 0.5 to 3.0 mmol/gDCW/h, over the growth rates from 0.05 to 0.3 1/h, respectively. Fluxes of PDC and PDH 
% were predicted to be from X to Y mmol/gDCW/h, representing Z % of the carbon from pyruvate branching point.”

% % from branching point means you take the flux OUT from pyruvate and divide against the flux IN to pyruvate (flux of enolase).
% model.metabolites.metabolite_ID.summary() gives the percentages

% For other sections, you might not have to go through this boring description, if it repeats. But for the first time, you must describe everything you plotted.

Model rhto-GEM estimates that the biomass growth rate, when glucose uptake is constrained to the experimentally measured specific glucose uptake rate, is lower compared to the experimental specific growth rate during the same glucose uptake rate, reaching only $0.25$ \unit{1/h} in simulations \textit{versus} $0.3$ \unit{1/h} experimentally. As the fluxes are visualized in the graphs over the growth rate, this is the reason why there are no predicted fluxes on growth rate over $0.25$ \unit{1/h}.  

Model predicted that exchange fluxes are higher than experimentally measured per growth rate (Figure \ref{rhto-GEM biomass max}.a). Model predicted that the fluxes of specific O$_2$ uptake and CO$_2$ secretion are $1.5-7.6$ and $1.6-8.4$ \unit{mmol/gDW/h} per growth rates from $0.03$ to $0.25$ \unit{1/h}, respectively. Whereas physiological data from experiments showed that the exchange fluxes of O$_2$ and CO$_2$ were $1.1-6.3$ and $1.2-7.4$ \unit{mmol/gDW/h}, per growth rates from $0.05$ to $0.3$ \unit{1/h} respectively. 
This result was expected as conventional GEMs (without resource allocation constraints, such as enzyme constraints) are known to overestimate exchange fluxes. 

All the investigated intracellular fluxes were predicted by the model to increase linearly over the predicted range of specific growth rates (\ref{rhto-GEM biomass max}.b). %growth rates from $0.03$ to $0.25$ \unit{1/h}.
Flux through glucose 6-phosphate dehydrogenase oxPPP was predicted to increase from $0.18$ to $1.45$ \unit{mmol/gDW/h}, representing around $48\%$ of the carbon from 
D-glucose 6-phosphate branching point. From the D-xylulose 5-phosphate branching point, $32\%$ of the flux was 
predicted to go to transketolase 1 (fluxes from $0.04$ to $0.34$ \unit{mmol/gDW/h}), $28\%$ to 
transketolase 2 ($0.04-0.3$ \unit{mmol/gDW/h}) and $40\%$ to phosphoketolase (fluxes from $0.05-0.4$ \unit{mmol/gDW/h}). 
The flux of transaldolase was predicted to be zero on all rates. Fructose biphosphate aldolase was predicted to increase from 
$0.28$ to $1.15$ \unit{mmol/gDW/h}, representing $82\%$ of the carbon from D-fructose 6-phosphate. 
Pyruvate decarboxylase represents $15\%$ and pyruvate dehydrogenase $59\%$ of the carbon from % This percent is a bit complicated bc there's mode mithocondrial pyruvate then transported there by pyruvate transport, 4% is produced by malic enzyme
cytososlic pyruvate branching point having fluxes from $0.06$ to $0.5$ and $0.46-2.25$ \unit{mmol/gDW/h}, respectively (Figure \ref{rhto-GEM biomass max}.b). For an overview, Escher \cite{King2015a} map visualizing the metabolic pathways was used and is accessible from Github \url{https://github.com/maivehanni/BSc_thesis/blob/main/Escher_maps/rhto_central_glc_uptake_1.1.png}.
The model did not predict any flux for ACL, instead it predicts that acetyl-CoA is produced through phosphoketolase. % for producing acetyl-CoA from citrate transported from mithocondria (TCA cycle), was zero. 
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{rhtoGEM_biomass_max.png}
    \caption{Simulated exchange (a) and intracellular (b) fluxes in \textit{R. toruloides} with model rhto-GEM optimized for biomass maximization and constrained over five specific glucose uptake rates. Exchange fluxes plot (a) shows the predicted specific glucose uptake (D-glucose exchange), specific oxygen consumption (O$_2$ exchange) and specific carbon dioxide production rate(CO$_2$ exchange). The experimentally measured exchange rates of oxygen (Specific O$_2$ consumption rate) and carbon dioxide (Specific CO$_2$ production rate) are visualized with dashed lines. Intracellular fluxes graph (b) shows the fluxes of glucose 6-phosphate dehydrogenase (oxPPP), transketolase 1, transaldolase, transketolase 2, fructose-biphosphate aldolase, pyruvate decarboxylase, pyruvate dehydrogenase, and in bold the fluxes of phosphoketolase and ATP-citrate lyase.}
    \label{rhto-GEM biomass max}
\end{figure}

For NADPH production and consumption there are no differences 
between different growth rates. In all cases the model predicts that around $90\%$ of the NADPH is produced by glucose 6-phosphate dehydrogenase and
phosphogluconate dehydrogenase (oxPPP). Ca $6\%$ is produced by methylenetetrahydrofolate dehydrogenase (Figure \ref{fig:rhtoGEM_bm_NADPH}). NADPH is primarily consumed by fatty acid synthases 50\% (fatty-acyl-CoA synthase (n-C16:0CoA) and fatty-acyl-CoA synthase (n-C18:0CoA)) and glutamate dehydrogenase by 30\%.  
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{rhtoGEM_bm_NADPH.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model rhto-GEM optimized for biomass maximization. Glucose uptake was constrained 
    to the lowest rate. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
    The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:rhtoGEM_bm_NADPH}
\end{figure}



\textbf{iRhtoC}

iRhtoC model predicts very similar exchange fluxes as the previous model, predicting higher O$_2$ and CO$_2$ rates than experimentally measured (Figure \ref{iRhtoC_biomass_max}.a). Model estimated that over the predicted range of specific growth rates, specific O$_2$ uptake and CO$_2$ production are $1.3-7.9$ and $1.4-8.8$ \unit{mmol/gDW/h}, respectively. Intracellular fluxes are predicted to increase lineraly over increasing growth rates. However, the predictions of intracellular fluxes by iRhtoC differ from model rhto-GEM. iRhtoC predicts that over the five growth rates the
flux through oxPPP increase from $0.24$ to $1.69$ \unit{mmol/gDW/h} (55\% from D-glucose 6-phosphate branching point), 
which is slightly more than rhto-GEM predicted.
Fluxes of transketolase 1 and 2 are also a bit higher than predicted by rhto-GEM ($0.08-0.56$ and $0.07-0.5$ \unit{mmol/gDW/h} representing 
53\% and 47\% from D-xylulose 5-phosphate branching point, respectively). 
Fluxes of transaldolase and pyruvate decarboxylase are predicted to be zero. 
Fructose-bisphosphate aldolase (representing 93\% from D-fructose 6-phosphate) and pyruvate dehydrogenase (80\% from cytososlic pyruvate 
branching point) 
fluxes are from $0.24$ to $1.49$ and from $0.53$ to $3.29$ \unit{mmol/gDW/h}, respectively.
ATP-citrate lyase has a flux from $0.18$ to $1.29$ \unit{mmol/gDW/h} representing 40\% of carbon from mithocondrial citrate. The flux of phosphoketolase is predicted to be zero (Figure \ref{iRhtoC_biomass_max}.b).
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{iRhtoC_biomass_max.png}
    \caption{Simulated exchange (a) and intracellular (b) fluxes in \textit{R. toruloides} with model iRhtoC optimized for biomass maximization 
    and constrained over five specific glucose uptake rates. Exchange fluxes plot (a) shows the predicted specific glucose uptake (D-glucose exchange), specific oxygen consumption (O$_2$ exchange) and specific carbon dioxide production (CO$_2$ exchange) rate. Intracellular fluxes graph (b) shows the fluxes of glucose 6-phosphate dehydrogenase (oxPPP), transketolase 1, transaldolase, transketolase 2, fructose-biphosphate aldolase, pyruvate decarboxylase, pyruvate dehydrogenase, and in bold the fluxes of phosphoketolase and ATP-citrate lyase.}
    \label{iRhtoC_biomass_max}
\end{figure}

This model predicts that almost $90\%$ of NADPH is produced by glucose 6-phosphate dehydrogenase and
phosphogluconate dehydrogenase (oxPPP) on all rates, but around $6\%$ is produced by malic enzyme on growth rates $0.03$ and $0.23$ \unit{mmol/gDW/h} and on other 
rates the model predicts isocitrate dehydrogenase instead (Figures \ref{fig:iRhtoC_bm_NADPH0} and \ref{fig:iRhtoC_bm_NADPH1}). 
Predicted consuming fluxes of NADPH do not differ between growth rates.
Similarly to model rhto-GEM, it is predicted that NADPH is consumed by glutamate dehydrogenase by 35\%. This model predicts lower use of fatty acid synthesis system (FAS), which is 25\%. (In this model, the enzymes of fatty acid synthesis system are distincted in more detail than in rhto-GEM, resulting in 16 FAS enzymes. Each respective enzyme's flux is around 1.5\% of total consumption flux of NADPH and they were included in the sector `Other consuming' in the pie chart, as 2.2\% was used as a cut off for including fluxes independently in the pie). 
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{iRhtoC_bm_NADPH_min.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model iRhtoC. The model was optimized for biomass maximization and glucose 
    uptake was constrained to the lowest rate. The fluxes are same when glucose uptake is constrained to highest rate. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes. The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:iRhtoC_bm_NADPH0}
\end{figure}
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{iRhtoC_bm_NADPH1.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R.toruloides} with model iRhtoC. The model was optimized for biomass maximization. Results are the same when glucose uptake is constrained to $1.114, 1.648$ or $2.305$ \unit{mmol/gDW/h}. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes. The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:iRhtoC_bm_NADPH1}
\end{figure}

\textbf{Rt\_IFO0880}

Rt\_IFO0880 also predicts higher exchange fluxes than experimentally measured (Figure \ref{fig:IFO0880_biomass_max}.a). Model predicted that over the predicted range of specific growth rates, specific O$_2$ uptake and CO$_2$ production fluxes are $0.5-3.1$, $1.1-6.8$ and $1.3-7.7$ \unit{mmol/gDW/h}, respectively. 
This model has two phosphoketolases, fructose-6-phosphate phosphoketolase (FPK) and xylulose-5-phosphate phosphoketolase (XPK), but as models rhto-GEM and iRhtoC have one phosphoketolase, FPK and XPK fluxes have been summed together for easier comparison with other models. Model predictions differ from models rhto-GEM and iRhtoC by not having any flux in oxPPP and transketolase 2. Transketolase 1 represents 13\% of the carbon from glyceraldehyde 3-phosphate. Transaldolase represents 19\%, fructose-bisphosphate aldolase 52\% and FPK 21\% carbon from D-glucose 6-phosphate. XPK represents 100\% of carbon from D-xylulose 5-phosphate. Summed flux of XPK and FPK is from $0.18$ to $1.25$ \unit{mmol/gDW/h} over the rates.
Pyruvate decarboxylase represents 7\% and pyruvate dehydrogenase 58\% of carbon from cytososlic pyruvate branching point (Figure \ref{fig:IFO0880_biomass_max}.b). The Escher map visualizing the pathways is accessible from \url{https://github.com/maivehanni/BSc_thesis/blob/main/Escher_maps/Rt_IFO0880_map_default.png}. Similarly to rhto-GEM, this model also predicts the use of phosphoketolase and no use of ACL. 
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{IFO0880_biomass_max.png}
    \caption{Simulated exchange (a) and intracellular (b) fluxes in \textit{R. toruloides} with model Rt\_IFO0880 optimized for biomass maximization and constrained over five specific glucose uptake rates. Exchange fluxes plot (a) shows the predicted specific glucose uptake (D-glucose exchange), specific oxygen consumption (O$_2$ exchange) and specific carbon dioxide production (CO$_2$ exchange) rate. Intracellular fluxes graph (b) shows the fluxes of glucose 6-phosphate dehydrogenase (oxPPP), transketolase 1, transaldolase, transketolase 2, fructose-biphosphate aldolase, pyruvate decarboxylase, pyruvate dehydrogenase, and in bold the fluxes of phosphoketolase and ATP-citrate lyase.}
    \label{fig:IFO0880_biomass_max}
\end{figure}

In NADPH production and consumption on there are no significant differences across the growth rates. However, around $90\%$ of NADPH is produced by alcohol dehydrogenase (ID: ALCD2y, Nicotinamide adenine dinucleotide phosphate[c] + Ethanol[c] => H+[c] + Nicotinamide adenine dinucleotide phosphate - reduced[c] + Acetaldehyde[c]) on all rates (Figure \ref{fig:IFO0880_biomass_max_NADPH_max}). This is very different from other models. 
On all rates, NADPH is consumed by glutamate dehydrogenase by around 35\% similarly to previous models. Around 7\% is consumed by fatty acyl CoA synthase n C80CoA lumped reaction and 10\% is the summed flux of five other fatty acyl CoA synthases (n C100CoA, n C120CoA, n C140CoA, n C160CoA and n C180CoA). The total flux of FAS is around 17\% and is comparable to iRhtoC.
%(That indicates unrealistic loops)
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{IFO0880_biomass_max_NADPH.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model Rt\_IFO0880. The model was optimized for biomass maximization 
    and glucose uptake was constrained. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
    The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:IFO0880_biomass_max_NADPH_max}
\end{figure}



\textbf{Rt\_IFO0880\_LEBp2023}

Exchange flux predictions are almost the same as with model Rt\_IFO0880. Model predicted that over the predicted range of specific growth rates, specific O$_2$ uptake and CO$_2$ production fluxes are $1.1-6.8$ and $1.3-7.7$ \unit{mmol/gDW/h}, respectively (Figure \ref{fig:Rt_IFO0880_LEBp2023_biomass_max}.a).
From D-glucose 6-phosphate branching point 9\% of the carbon is represented by oxPPP. 
Transketolase 1 and 2 represent very
low carbon percentage, only 0.3\% and 1.3\%, from glyceraldehyde 3-phosphate branching point, respectively. Transaldolase represents 0.6\%, transketolase 2 2\% and fructose-bisphosphate aldolase 89\% from D-glucose 6-phosphate. Pyruvate decarboxylase represents 5\% and pyruvate dehydrogenase 62\% carbon from cytososlic pyruvate. %cytosolic
33\% of carbon from mithocondrial citrate is represented by ACL. (\ref{fig:Rt_IFO0880_LEBp2023_biomass_max}.b)
The Escher map visualizing the pathways is accessible from \url{https://github.com/maivehanni/BSc_thesis/blob/main/Escher_maps/Rt_IFO0880_jsb_map_glc_uptake_1.1.png}.
On all growth rates the use of ACL is predicted instead of phosphoketolase, which is similar with the model iRhtoC. 
(As this model is an updated version of Rt\_IFO0880, it also has two phosphoketolases - FPK and XPK, which fluxes
have been summed together and represented as phosphoketolase.) 
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{Rt_IFO0880_LEBp2023_biomass_max.png}
    \caption{Simulated exchange (a) and intracellular (b) fluxes in \textit{R. toruloides} with model Rt\_IFO0880\_LEBp2023 optimized for biomass maximization and constrained over five specific glucose uptake rates. Exchange fluxes plot (a) shows the predicted specific glucose uptake (D-glucose exchange), specific oxygen consumption (O$_2$ exchange) and specific carbon dioxide production (CO$_2$ exchange) rate. Intracellular fluxes graph (b) shows the fluxes of glucose 6-phosphate dehydrogenase (oxPPP), transketolase 1, transaldolase, transketolase 2, fructose-biphosphate aldolase, pyruvate decarboxylase, pyruvate dehydrogenase, and in bold the fluxes of phosphoketolase and ATP-citrate lyase.}
    \label{fig:Rt_IFO0880_LEBp2023_biomass_max}
\end{figure}

On lowest growth rate, NADPH is predicted to be produced mainly ($\sim 85\%$) by aldehyde dehydrogenase (ALDD19xr, Phenylacetaldehyde[c] + H2O H2O[c] + Nicotinamide adenine dinucleotide[c] <=> 2 H+[c] + Nicotinamide adenine dinucleotide - reduced[c] + Phenylacetic acid[c]) and on all other rates mainly ($\sim 75\%$) by alcohol 
dehydrogenase and $\sim 15\%$ by glucose-6-phosphate dehydrogenase and phosphogluconate dehydrogenase (Figures \ref{fig:Rt_IFO0880_LEBp2023_biomass_max_NADPH0} and \ref{fig:Rt_IFO0880_LEBp2023_biomass_max_NADPH1}). 
Interestingly, as the growth rate increases the use of aldehyde dehydrogenase slightly decreases and use of oxidative pentose phosphate pathway for NADPH production increases. 
On all growth rates, NADPH is primarily consumed by glutamate dehydrogenase (35\%) and FAS (20\%) (fatty acyl CoA synthase n C80CoA lumped reaction, fatty acyl CoA synthase n C100CoA, C120CoA, C140CoA and C160CoA).
% (That indicates unrealistic loops?)
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{Rt_IFO0880_LEBp2023_biomass_max_NADPH.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model Rt\_IFO0880\_LEBp2023. The model was optimized for biomass maximization. 
    Glucose uptake was constrained on the lowest rate. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
    The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:Rt_IFO0880_LEBp2023_biomass_max_NADPH0}
\end{figure}
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{Rt_IFO0880_LEBp2023_biomass_max_NADPH1.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model Rt\_IFO0880\_LEBp2023. The model was optimized for biomass maximization 
    and glucose uptake constrained to $1.114$. 
    The results are very similar when glucose uptake is constrained to $1.114, 1.648, 2.305$ or $3.1$. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
    The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:Rt_IFO0880_LEBp2023_biomass_max_NADPH1}
\end{figure}

Simulations with biomass maximisation as an objective function under carbon limitation, with glucose as the sole source of carbon, showed that the models prefer different pathways for 
production of acetyl-CoA. The models rhto-GEM and Rt\_IFO0880 produced acetyl-CoA through phosphoketolase, whereas models iRhtoC and the modified model 
of Rt\_IFO0880 (Rt\_IFO0880\_LEBp2023) predicted the production of acetyl-CoA using ACL. In many studies, ACL has been found to be present in oleaginous yeast, whereas phosphoketolase has been predicted to be active in \textit{R. toruloides} during growth on xylose by \cite{Pinheiro2020, Rekena2023} and on glucose by \cite{Rekena2023}, but in both studies protein levels of ACL were also reported and in \cite{Pinheiro2020} ACL was found in even higher levels under nitrogen limitation, when lipid accumulation was increased.  % MAybe add nmore what has been found in these studies
In experiments where phosphoketolase pathways were transferred to the oleaginous model yeast \textit{Y. lipolytica}, the lipid production in the engineered yeast increased by 53\% when grown on glucose \cite{Xu2016}.

NADPH sources also differ among model predictions. On all growth rates, models rhto-GEM and iRhtoC predict that most of the NADPH is produced by the oxidative pentose phosphate pathway and $\sim 6\%$ by ethylenetetrahydrofolate dehydrogenase in rhto-GEM and by malic enzyme in iRhtoC. Model Rt\_IFO0880 predicts that most of the NADPH is produced by alcohol dehydrogenase and Rt\_IFO0880\_LEBp2023 predicts that on the lowest growth rate NADPH is mainly ($\sim 85\%$) produced through aldehyde dehydrogenase and on all other rates, $\sim 75\%$ is produced by alcohol dehydrogenase and $\sim 15\%$ by oxPPP. Malic enzyme, which is considered a key enzyme in the regeneration of NADPH for lipid biosynthesis \cite{Ratledge2002}, was not predicted to be the primary NADPH producer in any of the models. However, this is in accordance with the proteomics data of \textit{R. toruloides} grown on xylose by \cite{Pinheiro2020}, where it was predicted by the rhto-GEM that most of the NADPH is produced in oxPPP. It was also suggested by Wasylenko et al. 2015 that in \textit{Yarrowia lipolytica} the oxidative pentose phosphate pathway is the primary source of NADPH during lipid accumulation on growth on glucose \cite{Wasylenko2015}. Still, the results by Li et al. 2012 indicated that the lipid content in \textit{Rhodotorula glutinis} grown on glucose increased from 18.7\% to 39.4\% thanks to overexpression of malic enzyme, indicating importance of ME \cite{Li2012}. These results in this study as well as in others, demonstrate the carbon source-dependent differences in yeast metabolism.

% Predicted consuming fluxes inidicated that in model rhto-GEM NADPH is primarily consumed by fatty acid synthases and glutamate dehydrogenase, this is in accordance with the predictions with an enzyme-constrained \textit{R. toruloides} model by \cite{Rekena2023}.


\section{Non-growth associated maintenance minimisation as an objective function}

To see whether different objctive function changes the flux patterns, metabolic flux patterns were further investigated using the minimisation of non-growth associated maintenance (NGAM) reaction as an objective function. 
Specifically, whether Rt\_IFO0880-based models could re-arrange NADPH regeneration through different pathways than alcohol and aldehyde dehydrogenase. When optimizing for NGAM minimisation, it is assumed that cells strive for satisfying physiological parameters with least energy expenditure as NGAM minimisation decreases the ATP demand.
% Can you add one sentence on what exactly this obj. function does? Optimizing for.. under assumption that cell 
% will strive for satisfying physiological parameters with least energy expenditure?

Same experimentally measured specific glucose uptake rates were used throughout the simulations (Table \ref{table:LabData}). This objective function also needed constraints on biomass growth rate because otherwise the simulation chooses zero as its flux. Experimentally measured specific growth rates together with experimental specific glucose uptakes were infeasible for the models. Because of that, growth rate was constrained to the values that each model predicted in the simulations optimized for biomass maximisation. These values slightly varied between the models, but when rounded were 0.03, 0.08, 0.12-0.13, 0.17-0.18 and 0.23-0.25 \unit{1/h}. All simulated fluxes are available on a Github repository (\url{https://github.com/maivehanni/BSc_thesis/tree/main/All_simulated_fluxes}). 

% \textbf{rhto-GEM} - but remember the results for defence

% Model rhto-GEM optimized for NGAM minimisation predicts similar fluxes as with 
% previous objective function. 

% \textbf{iRhtoC} - but remember for defence

% iRhtoC optimized for NGAM minimisation predicts same exchange and intracellular fluxes as with 
% previous objective function. 
% As previously, most of the NADPH is produced by oxPPP, but on the lowest growth rate $\sim 6\%$ is produced by isocitrate dehydrogenase and on all other 
% rates by malic enzyme (figures \ref{fig:iRhtoC_nm_NADPH_min} and \ref{fig:iRhtoC_nm_NADPH1}). NGAM minimisation and biomass maximisation as an objective 
% function predict differences in the use of malic enzyme vs isocitrate dehydrogenase on different growth rates.

% \begin{figure}[H]
%     \centering
%     \includegraphics[width=\linewidth]{iRhtoC_nm_NADPH_min.png}
%     \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model iRhtoC. The model was optimized for NGAM minimization. 
%     Glucose uptake and growth rate were constrained on the lowest rate. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
%     The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
%     that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
%     \label{fig:iRhtoC_nm_NADPH_min}
% \end{figure}
% \begin{figure}[H]
%     \centering
%     \includegraphics[width=\linewidth]{iRhtoC_nm_NADPH1.png}
%     \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model iRhtoC. The model was optimized for NGAM minimization, 
%     growth rate and glucose uptake were constrained to $0.08$ and $1.114$ \unit{mmol/gDW/h}, respectively. 
%     The results are the same when glucose uptake is constrained to $1.114, 1.648, 2.305$ or $3.1$ \unit{mmol/gDW/h}. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
%     The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
%     that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
%     \label{fig:iRhtoC_nm_NADPH1}
% \end{figure}
% \textbf{Rt\_IFO0880}
As a result, model Rt\_IFO0880 optimized for NGAM minimization predicted the production of NAPDH differently - around $90\%$ of NADPH is produced by homoserine dehydrogenase on rates $0.08$ and $0.17$ \unit{1/h} and on other rates by alcohol dehydrogenase (Figures \ref{fig:IFO0880_nm_NADPH} and \ref{fig:IFO0880_nm_NADPH1}). NADPH consuming fluxes did not change, like previously, majority is being consumed by glutamate dehydrogenase and 17\% by FAS.  

\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{IFO0880_nm_NADPH.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model Rt\_IFO0880. The model was optimized for NGAM minimization. 
    Glucose uptake and growth rate were constrained on the lowest rate. The results are the same when glucose uptake is constrained to $1.648$ or $3.1$ \unit{mmol/gDW/h}. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
    The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:IFO0880_nm_NADPH}
\end{figure}
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{IFO0880_nm_NADPH1.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model Rt\_IFO0880. The model was optimized for NGAM minimization, 
    growth rate and glucose uptake were constrained to $0.08$ \unit{1/h} and $1.114$ \unit{mmol/gDW/h}, respectively. The results are similar when glucose uptake is constrained to $2.305$ \unit{mmol/gDW/h}. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
    The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:IFO0880_nm_NADPH1}
\end{figure}

% \textbf{Rt\_IFO0880\_LEBp2023}

Model Rt\_IFO0880\_LEBp2023 optimized for NGAM minimization also predicted the production of NAPDH differently compared to the simulation with biomass maximization as the objective function. On rates $0.08, 0.12$ and $0.17$ \unit{1/h} use of homoserine dehydrogenase is predicted instead of alcohol dehydrogenase (Figure \ref{fig:Rt_IFO0880_LEBp2023_nm_NADPH1}). 
On lowest rate aldehyde dehdrogenase and on highest rate alcohol dehydrogenase is predicted, which is the same as previously, when biomass 
maximisation was used as an objective function. NADPH consuming fluxes do not differ from previous simulation results: 35\% being consumed by glutamate dehydrogenase and 20\% by FAS.  
\begin{figure}[H]
    \centering
    \includegraphics[width=\linewidth]{Rt_IFO0880_LEBp2023_nm_NADPH1.png}
    \caption{Simulated NADPH producing and consuming fluxes in \textit{R. toruloides} with model Rt\_IFO0880\_LEBp2023. The model was optimized for NGAM minimization, 
    growth rate and glucose uptake were constrained to $0.08$ \unit{1/h} and $1.114$ \unit{mmol/gDW/h}, respectively. The results are very similar when glucose uptake is constrained to $1.648$ or $2.305$ \unit{mmol/gDW/h}. The upper part of the pie shows producing fluxes and the lower part shows consuming fluxes.
    The flux of the enzyme is shown between the brackets after the name of the metabolite, and infront of the name is the percent
    that the given enzyme makes up of the total producing or consuming NADPH flux, respectively.}
    \label{fig:Rt_IFO0880_LEBp2023_nm_NADPH1}
\end{figure}

All in all, these simulations showed that Rt\_IFO0880-based models could re-arrange NADPH regeneration through different pathways than alcohol and aldehyde dehydrogenase when NGAM minimization is used as an objective function. However, the predictions did not lead towards the oxidative part of pentose phosphate pathway, but instead, to homoserine dehydrogenase for majority of simulated biomass production rates (on $0.08$ and $0.17$ \unit{1/h} for Rt\_IFO0880 and on $0.08, 0.12$ and $0.17$ \unit{1/h} for Rt\_IFO0880\_LEBp2023). This result was unexpected and it is not clear what made the models predict alcohol and aldehyde dehdrogenase only on some growth rates.

% % Discussion
% Still, the autohor of the enhanced model of Rt\_IFO0880 named Rt\_IFO0880\_LEBp2023, has pointed out that the improvements that resulted in the Rt\_IFO0880\_LEBp2023 
% model cover only a small portion of the metabolism of \textit{R. torluoides}. "The differences are possibly the "tip of the iceberg" of many others and
% the differences between the existing GEMs of \textit{R. toruloides} are significant and may lead to different understandings of
% this yeast's physiology if used alone" \cite{DeBiaggi2023}.

%  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9560524/ an article were dif obj funcs were compared




